openpype/tools/loader/widgets.py

```diff
@@ -1075,9 +1088,21 @@ def get_enabled_families(self):
             if checked == QtCore.Qt.Checked:
                 family = index.data(QtCore.Qt.DisplayRole)
                 checked_families.append(family)
-
         return checked_families
 
+    def set_last_families(self, families):
+        if not families:
+            return
+
```

Comment:
Is this missing self.blockSignals(True)`  since we're setting it to false at the end?